lua/lvim/lsp/handlers.lua

```diff
@@ -3,138 +3,13 @@
 local M = {}
 
 function M.setup()
-  local config = { -- your config
-    virtual_text = lvim.lsp.diagnostics.virtual_text,
-    signs = lvim.lsp.diagnostics.signs,
-    underline = lvim.lsp.diagnostics.underline,
-    update_in_insert = lvim.lsp.diagnostics.update_in_insert,
-    severity_sort = lvim.lsp.diagnostics.severity_sort,
-    float = lvim.lsp.diagnostics.float,
-  }
-  if vim.fn.has "nvim-0.6" == 1 then
-    vim.diagnostic.config(config)
-  else
-    vim.lsp.handlers["textDocument/publishDiagnostics"] = function(_, _, params, client_id, _)
-      local uri = params.uri
-      local bufnr = vim.uri_to_bufnr(uri)
-      if not bufnr then
-        return
-      end
-
-      local diagnostics = params.diagnostics
-      vim.lsp.diagnostic.save(diagnostics, bufnr, client_id)
-      if not vim.api.nvim_buf_is_loaded(bufnr) then
-        return
-      end
-      vim.lsp.diagnostic.display(diagnostics, bufnr, client_id, config)
-    end
-
-    vim.lsp.handlers["textDocument/hover"] = vim.lsp.with(vim.lsp.handlers.hover, {
-      border = lvim.lsp.popup_border,
```

Comment:
We need to remove the corresponding option as well, see https://github.com/LunarVim/LunarVim/issues/2095

Reply:
I was actually wrong, sorry 😢 
We need to restore these along with all the lsp options.

Reply:
🚬

Reply:
we can merge your PR on top of this